src/app/services/auth.service.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { AuthService } from './auth.service';
+import { TestBed, inject } from '@angular/core/testing';
+import { OAuthService } from 'angular-oauth2-oidc';
+
+describe('AuthService', () => {
+  const fakeToken = {};
+  const fakeRoles = ['Administrators'];
+  const fakeRoleWithoutUsers = ['Foo'];
+
+  beforeEach(() => {
+     TestBed.configureTestingModule({
+       providers: [
+        AuthService,
+        {provide: OAuthService, useValue: {
+          getIdentityClaims: () => ['Administrators']
```

Comment:
use the fakeRoles const here instead of creating the array again

Reply:
Will do.